ArkKit/ark-render-kit/ArkCanvasSystem.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class ArkCanvasSystem: System {
+    var active: Bool
+    static let canvasComponentTypes: [CanvasComponent.Type] = [
+        ButtonCanvasComponent.self,
+        JoystickCanvasComponent.self,
+        CircleCanvasComponent.self,
+        RectCanvasComponent.self,
+        PolygonCanvasComponent.self,
+        BitmapImageCanvasComponent.self
+    ]
+    init(active: Bool = true) {
+        self.active = active
+    }
+    func update(deltaTime: TimeInterval, arkECS: ArkECS) {
+        let entitiesWithCanvas = arkECS.getEntities(with: ArkCanvasSystem.canvasComponentTypes)
```

Comment:
`.getEntities(with: )` uses intersection, not a logical OR

Reply:
thanks, good catch, i'd fix